src/vicsetup/todo.txt

```diff
@@ -0,0 +1,17 @@
+    AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
```

Comment:
It would be good to open Github issues instead.

Reply:
Resolved (I added issues and deleted this file)